nesc/whip6/platforms/boards/cc26xxbased/cherry-v2/ICountPub.nc

```diff
@@ -12,7 +12,7 @@ generic configuration ICountPub() {
 }
 
 implementation {
-    components new HalEventCountPub(60 * 1024) as EvCnt;
```

Comment:
General questions:
- why two timer channels?
- why replacing the existing module? why not creating a second implementation instead?

Reply:
1. because we have two timer channels, so why not handle them all
2. I can do that if you want